base/src/consensus_state.rs

```diff
@@ -81,9 +86,129 @@ pub struct ConsensusState {
     pub supercharge_coinbase: bool,
 }
 
+/// For ConsensusConstants Default
+pub struct ConsensusConstants {
+    /// Point of finality (number of confirmations)
+    pub k: Length,
+    /// Number of slots per epoch
+    pub slots_per_epoch: Length,
+    /// No of slots in a sub-window = 7
+    pub slots_per_sub_window: Length,
+    /// Maximum permissable delay of packets (in slots after the current)
+    pub delta: Length,
+    /// Timestamp of genesis block in unixtime
+    pub genesis_state_timestamp: BlockTime,
+    /// Sub windows within a window
+    pub sub_windows_per_window: Length,
+    /// Number of slots before minimum density is used in chain selection
+    pub grace_period_end: Length,
+}
+
+impl Default for ConsensusConstants {
+    fn default() -> Self {
+        // TODO: read from config
```

Comment:
let's open an issue for this if there isn't one

Reply:
Opend [#127 ](https://github.com/ChainSafe/mina-rs/issues/127)